scripts/deploy.sh

```diff
@@ -31,16 +31,15 @@ function initialize {
 function upload_assets {
   git clone --quiet --depth 1 git@github.com:${SITE_REPO_SLUG}.git "$SITE_HOME"
   mv -f "$TLDR_ARCHIVE" "$SITE_HOME/assets/"
-  mv -f "${TLDR_LANG_ARCHIVES_DIRECTORY}"/*.zip "$SITE_HOME/assets/"
+  find "$TLDR_LANG_ARCHIVES_DIRECTORY" -maxdepth 1 -name "*.zip" -exec mv -f {} "$SITE_HOME/assets/" \;
```

Comment:
Need a single quote on `-name` here to be completely robust.

```suggestion
  find "$TLDR_LANG_ARCHIVES_DIRECTORY" -maxdepth 1 -name '*.zip' -exec mv -f {} "$SITE_HOME/assets/" \;
```